R/SurveyFit.R

```diff
@@ -142,59 +150,237 @@ SurveyFit <- R6::R6Class(
       } else {
         out <- data.frame(value = apply(poststrat_estimates, 2, function(x) sum(poststrat_data$N_j*x)/sum(poststrat_data$N_j)))
       }
+      rownames(out) <- NULL
       out
     },
-    #' @description Plot takes the aggregated estimates and produces a quick visualization total and sub-population estimates.
-    #' @param aggregated_estimates The object returned by `aggregate`
-    #' @param weights TRUE (default) if weighted estimates are included for comparison. Weighted interval is a 95% interval.
-    #'  If no weights are specified, weights are assumed to be 1, which is roughly equivalent to the observed data average.
-    plot = function(aggregated_estimates, weights = TRUE) {
+    #' @description Creates a set of summary statistics of the mrp estimate,
+    #' and corresponding weighted and raw data estimates
+    #' @param aggregated_estimates The data frame returned by the `aggregate() method`.
+    #' @return A data frame. If `by` is not specified then the data frame will
+    #'   have number of rows equal to the number of posterior draws. If `by` is
+    #'   specified the data frame will have number of rows equal to the number
+    #'   of posterior draws times the number of levels of the `by` variable,
+    #'   and there will be an extra column indicating which level of the `by`
+    #'   variable each row corresponds to.
```

Comment:
this looks like the doc for the returned data frame from aggregate(). i think it needs to be updated to describe the output from summary() instead.